ML-Frameworks/pytorch-aarch64/build.sh

```diff
@@ -28,7 +28,7 @@ if [ -d pytorch ] || [ -d builder ] || [ -d ComputeLibrary ] ; then
     echo "You appear to have sources already (pytorch/builder/ComputeLibrary)" \
 
     if ! ([[ $* == *--force* ]] || [[ $* == *--use-existing-sources* ]]) ; then
-        >2& echo "rerun with --force to overwrite sources or with" \
+        >&2 echo "rerun with --force to overwrite sources or with" \
```

Comment:
I could have sworn this had been fixed!

Reply:
Yes, I previously had to do it for tensorflow here https://github.com/ARM-software/Tool-Solutions/pull/312

I must have missed this.